mycore-base/src/main/java/org/mycore/resource/common/MCRResourceUtils.java

```diff
@@ -37,4 +42,24 @@ public static URL toFileUrl(Path path) {
         }
     }
 
+    public static URL toJarFileUrl(String jarPath, String path) {
+        return toJarFileUrl(Path.of(jarPath), Path.of(path));
+    }
+
+    public static URL toJarFileUrl(Path jarPath, String path) {
+        return toJarFileUrl(jarPath, Path.of(path));
+    }
+
+    public static URL toJarFileUrl(String jarPath, Path path) {
+        return toJarFileUrl(Path.of(jarPath), path);
+    }
+
+    public static URL toJarFileUrl(Path jarPath, Path path) {
+        try {
+            return URI.create("jar:" + toFileUrl(jarPath) + "!" + path.toAbsolutePath()).toURL();
```

Comment:
this adds a "C:" to the path inside the jar behind the "!"

Reply:
should be fixed now